meshroomPlugin.json

```diff
@@ -0,0 +1,45 @@
+[
+	{
```

Comment:
Formatting issue: a tab '\t' is used as indentation before the first element, whereas spaces are used in the rest of the document.
Fix suggestion: replace the tab with spaces to preserve consistency.